app/workers/work_packages/apply_working_days_change_job.rb

```diff
@@ -34,6 +34,10 @@ class WorkPackages::ApplyWorkingDaysChangeJob < ApplicationJob
     total_limit: 1
   )
 
+  retry_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError,
+           wait: 5.minutes,
+           attempts: 3
+
```

Comment:
With a `total_limit` of 1, the ConcurrencyExceededError should never happen I believe. Because it would only be thrown for a queued job that tries to be executed. But it shouldn't even be possible to queue a second job.